SH4RE/SH4RE/CreateListing/CreateListingView.swift

```diff
@@ -226,6 +224,20 @@ struct CreateListingView: View {
             }
             else {
                 calAvail.append(availabilitySelection)
+                let calendar = Calendar.current
+                let components = calendar.dateComponents([.year, .month, .day], from: Date())
+                let startOfMonth = calendar.date(from:components)!
+                let numberOfDays = calendar.range(of: .day, in: .quarter, for: startOfMonth)!.upperBound
+                let allDays = Array(0..<numberOfDays).map{ calendar.date(byAdding:.day, value: $0, to: startOfMonth)!}
+                if (availabilitySelection == "Everyday") {
+                    calAvail = []
+                }
+                else if (availabilitySelection == "Weekdays") {
+                    calAvail = allDays.filter{ calendar.isDateInWeekend($0) }
```

Comment:
why are we filtering the dates that ARE in the weekend, if the availability is set to weekdays?

Reply:
it's just cause of the way it's handled on view listing side. The dates that it retrives is the dates to **block**, so if we want weekdays we want to store weekends in database cause those will be blocked when creating availbility during view listing